test/testdrive/mzcompose.yml

```diff
@@ -8,6 +8,58 @@
 # by the Apache License, Version 2.0.
 
 version: '3.7'
+
+mzworkflows:
+  # Run tests, requires aws credentials in the environment
+  ci:
+    env:
+      AWS_REGION: us-east-2
+      AWS_ENDPOINT: ''
+      # The first TD_TEST parameter is automatically appended to
+      # test/testdrive, but the second is just a plain string
+      TD_TEST: ${TD_TEST:-*.td test/testdrive/esoteric/*.td}
+    steps:
+      - step: workflow
+        workflow: start-deps
+      - step: run
+        service: testdrive
+
+  # Run tests with no support for s3
+  local:
+    steps:
+      - step: workflow
+        workflow: start-deps
+      - step: run
+        service: testdrive
+
+  # Run tests with s3 stubbed out, no credentials needed
+  local-aws:
+    env:
+      AWS_REGION: localstack
+      AWS_ENDPOINT: http://localstack:4566
+    steps:
+      - step: start-services
+        services: [localstack]
+      - step: workflow
+        workflow: start-deps
+      - step: wait-for-tcp
+        host: localstack
+        port: 4566
+      - step: run
+        service: testdrive
+
+  start-deps:
+    steps:
+      - step: start-services
+        services: [kafka, schema-registry, materialized]
+      - step: wait-for-tcp
+        host: schema-registry
+        port: 8081
+      - step: wait-for-tcp
+        host: kafka
+        port: 9092
+      - step: wait-for-mz
```

Comment:
This seems like duplicate waiting. If we do this, can we get rid of the `wait-for-it` commands in the entrypoint, and also in the `testdrive` image?

Reply:
yes, we can and I'd prefer it, I just didn't want to bust anyone's workflow. That said, I can just make an announcement.

Reply:
Oh, you know, the other thing we could do is rename `local` to `testdrive`, and rename the `testdrive` service to `testdrive-container` or something like that.